docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -34,21 +36,42 @@ const statusMap: Map<UserStatus, StatusPropsExtendable> = new Map([
 function generateChatMsgProps(msg: MessageData, fromUser: UserData): ChatMessage {
   const { content, mine } = msg
   const msgProps: ChatMessage = {
-    content: msg.withAttachment
-      ? { content: createMessageContentWithAttachments(content) }
-      : content,
+    // aria-labelledby will need to by generated based on the needs. Currently just hardcoded.
+    'aria-labelledby': `sender-${msg.id} timestamp-${msg.id} content-${msg.id}`,
```

Comment:
lets avoid `msg` shortcuts, as there is no need for them (and, also, this would follow our general principle to avoid shortcuts)